### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by vaishnavipaliya11**
Authentication is added 
Guest login is created
Sign up functionality is added.
image

### CODE REVIEW

File: src/pages/LoginPage/Login.js [hunk 1]
```diff
@@ -1,25 +1,84 @@
-import React from 'react'
-import { Link } from 'react-router-dom'
-import "./Login.css"
+import React from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { useState } from "react";
```
Comment (shraddha15-creator) [inline src/pages/LoginPage/Login.js:3]: Import useState at !st line --> 
` import React, { useState } from "react"; `

File: src/pages/LoginPage/Login.js [hunk 2]
```diff
@@ -1,25 +1,84 @@
...
+const Login = () => {
+  const [userDetails, setUserDetails] = useState({ email: "", password: "" });
+  const { auth, setAuth } = useAuth();
+  const navigate = useNavigate();
+
+  const onChangehandler = (e) => {
+    const { name, value } = e.target;
+    setUserDetails({ ...userDetails, [name]: value });
+
+    console.log(userDetails);
```
Comment (shraddha15-creator) [inline src/pages/LoginPage/Login.js]: remove console

**Review by shraddha15-creator — APPROVED** [review]
Code looks clean and readable.

File: src/pages/SignUpPage/SignUp.js [hunk 3]
```diff
@@ -1,33 +1,125 @@
...
+    Email: "",
+    Password: "",
+  });
+
+  const handelUserInput = (e) => {
+    const { name, value } = e.target;
+    setUserData({ ...userData, [name]: value });
+
+    setErrorMsg(false);
+    console.log(userData);
```
Comment (deepak29-git) [inline src/pages/SignUpPage/SignUp.js]: remove console

**Review by deepak29-git — APPROVED** [review]
variable name is good

**Pull request merged**